tests/installation/installation_overview.pm

```diff
@@ -62,6 +62,30 @@ sub ensure_ssh_unblocked {
     }
 }
 
+sub check_default_target {
+    # Check the systemd target where scenario make it possible
+    return if (is_caasp || is_upgrade || is_hyperv ||
+        get_var('REMOTE_CONTROLLER') || (get_var('BACKEND', '') =~ /spvm|pvm_hmc/));
+    # exclude non-desktop environment as well
+    return unless get_var('DESKTOP');
+
+    # Set expectations
+    my $expected_target = check_var('DESKTOP', 'textmode') ? "multi-user" : "graphical";
+
+    select_console 'install-shell';
+
+    # default.target is not yet linked, so we have to parse the logs
+    my $target_search = 'default target has been set';
+    my $log_line      = script_output("grep '$target_search' /var/log/YaST2/y2log | tail -1");
```

Comment:
As per our discussion, as this line theoretically can be in wrapped logs, what's about using zgrep, e.g.:
`zgrep -e 'bla'  /var/log/YaST2/y2log*`?

Reply:
the log rotation never happened at that point of the installation (it was another issue at the end) so we are safe. I'm afraid that zgrep might not be available in some flavor as this test is wide used.

Reply:
Good point, we might not have zgrep everywhere, but still to play it safe I would not kill the test if we failed to find target in the logs.

Reply:
exactly, I fixed that according to your other suggestion below.